src/test/java/net/sf/jabref/importer/fileformat/BibtexParserTest.java

```diff
@@ -1486,4 +1487,43 @@ public void parseReturnsEntriesInSameOrder() throws IOException {
 
         assertEquals(expected, result.getDatabase().getEntries());
     }
+
+    @Test
+    public void parsePrecedingComment() throws IOException {
+        // @formatter:off
+        String bibtexEntry = "% Some random comment that should stay here" + Globals.NEWLINE +
```

Comment:
What happens to some random test in front of the entry? Bib(la)tex does not know the percentage symbol as a comment symbol...